analyses/tcga-capture-kit-investigation/scripts/intersect-bed-maf.sh

```diff
@@ -15,14 +15,24 @@ SCRATCH='../../scratch'
 
 
 ## get intersection between all the BED files
-bedtools intersect -a $DATA/WGS.hg38.strelka2.unpadded.bed -b $DATA/WGS.hg38.mutect2.vardict.unpadded.bed \
-| bedtools intersect -a - -b $DATA/WGS.hg38.lancet.unpadded.bed \
-| bedtools intersect -a - -b results/whole_exome_agilent_1.1_refseq_plus_3_boosters.targetIntervals.Gh38.bed \
-| bedtools intersect -a - -b results/whole_exome_agilent_designed_120.targetIntervals.Gh38.bed \
-| bedtools intersect -a - -b results/whole_exome_agilent_plus_tcga_6k.targetIntervals.Gh38.bed \
-| bedtools intersect -a - -b results/tcga_6k_genes.targetIntervals.Gh38.bed \
-> $SCRATCH/intersect-all-tcga_pbta.bed
+# bedtools intersect -a $DATA/WGS.hg38.strelka2.unpadded.bed -b $DATA/WGS.hg38.mutect2.vardict.unpadded.bed \
+# | bedtools intersect -a - -b $DATA/WGS.hg38.lancet.unpadded.bed \
+# | bedtools intersect -a - -b results/whole_exome_agilent_1.1_refseq_plus_3_boosters.targetIntervals.Gh38.bed \
+# | bedtools intersect -a - -b results/whole_exome_agilent_designed_120.targetIntervals.Gh38.bed \
+# | bedtools intersect -a - -b results/whole_exome_agilent_plus_tcga_6k.targetIntervals.Gh38.bed \
+# | bedtools intersect -a - -b results/tcga_6k_genes.targetIntervals.Gh38.bed \
+# > $SCRATCH/intersect-all-tcga_pbta.bed
 
+## make the bedtools intersect more general use
+## set base intersect bedtools cmd to get PBTA intersection
+BEDTOOLS_CMD="""bedtools intersect -a $DATA/WGS.hg38.strelka2.unpadded.bed -b $DATA/WGS.hg38.mutect2.vardict.unpadded.bed \
+| bedtools intersect -a - -b $DATA/WGS.hg38.lancet.unpadded.bed"""
+## loop all liftover'd TCGA BED from ../results/ folder and add on to the cmd
+# ls bedfiles/*.Gh38.bed | while read tcga_bedfile
+for tcga_bedfile in results/bedfiles/*
```

Comment:
Preserve a bit of specificity, in case something goes wrong
```suggestion
for tcga_bedfile in results/bedfiles/*.bed
```